src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -193,4 +193,9 @@ object FireCloudConfig {
     val entityWorkspaceName: Option[String] = if (metrics.hasPath("entityWorkspaceName")) Some(metrics.getString("entityWorkspaceName")) else None
     val libraryNamespaces: List[String] = metrics.getStringList("libraryWorkspaceNamespace").asScala.toList
   }
+
+  object Notification {
+    private val notification = config.getConfig("notification")
```

Comment:
i can't find the fc-develop branch with the config change here, and it's causing orch to not come up in my phase4 fiab. any clues? @dvoet @MatthewBemis